content/01.proposal.md

```diff
@@ -0,0 +1,54 @@
+## Proposal {.page_break_before}
+
+Manubot is a system for writing scholarly manuscripts on GitHub.
+Manubot aims to **transform publishing** to be transparent & reproducible, immediate & permissionless, versioned & automated, collaborative & open, linked & provenanced, decentralized & hackable, interactive & annotated, and free of charge.
+Wooh!
```

Comment:
I'm not sure this helps you out here. 😉

Reply:
https://github.com/greenelab/manufund-2018/pull/2/commits/fd5747d4f2979a3fdd3e27450c398c3b4d6d3af4